docs/automated-tests/test-execution/quick-run-timeouts.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Execution Timeouts in Quick Execution
 page_title: Execution Timeouts in Quick Execution
-description: "Test Studio Quick test Execution. Test Studio test run. annotated test run. Global Timeouts in Test Studio project. Visual debugger options in Test Studio. Set preferred browser for test runs and test recording. Quick run Execution log "
+description: "Learn how to adjust the execution timeouts for a quick test execution in Test Studio."
 position: 4
 ---
 # Adjust the Execution Timeouts
 
-The **Test Ribbon** in Test Studio allows you to trigger a quick test execution. Along with that it provides quick access to some useful project settings.
+The **Test** ribbon gives you quick access to the following timeout settings:
 
-The **Wait on elements** timeout sets the amount of time, which will be waited for an element to exist on a page. When the set time passes and the element cannot be located on the page, the test will fail with _'Element not found'_ error. The default value is 15 seconds and depending on the application overall performance, you may need to increase/decrease it.
```

Comment:
- _Always preserve the capitalization used by the UI. See https://testdocs.telerik.com/style-guide/element-formatting#guidelines
- _Sets the amount of time, which will be waited_ - Abuse of passive voice. See https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-the-active-voice